jwql/website/apps/jwql/forms.py

```diff
@@ -187,10 +239,8 @@ class AnomalyQueryForm(BaseForm):
     instrument = forms.MultipleChoiceField(required=False,
                                            choices=[(inst, JWST_INSTRUMENT_NAMES_MIXEDCASE[inst]) for inst in JWST_INSTRUMENT_NAMES_MIXEDCASE],
                                            widget=forms.CheckboxSelectMultiple)
-    aperture = forms.MultipleChoiceField(required=False, choices=aperture_list, widget=forms.CheckboxSelectMultiple)
-    filt = forms.MultipleChoiceField(required=False, choices=filter_list, widget=forms.CheckboxSelectMultiple)
-    early_date = forms.DateField(required=False, initial="eg, 2021-10-02 12:04:39 or 2021-10-02")
-    late_date = forms.DateField(required=False, initial="eg, 2021-11-25 14:30:59 or 2021-11-25")
+    # early_date = forms.DateField(required=False, initial="eg, 2021-10-02 12:04:39 or 2021-10-02")
+    # late_date = forms.DateField(required=False, initial="eg, 2021-11-25 14:30:59 or 2021-11-25")
```

Comment:
Should these be un-commented?

Reply:
I was thinking of adding these back in (along with the gratings) if there were a handful of additional parameters that instrument teams wanted to sort by. It looks a little clunky right now with a form row with just one field, so they aren't actually in the form at the moment. Do you think I should add those and the gratings (and any other parameters) back in for this round? I can also delete the lines for now and add it back later.

Reply:
I'm not sure that it matters too much either way, but my hunch is that we should keep it simple for now and only add more parameters as teams ask for them.